### DISCUSSION TITLE
How can I specify the compressors only for the build command

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [parcel-bundler/parcel#7745] [Q&A] [answered]
**How can I apply the compression only to the build command so that `npm run build` creates only gzipped files and `npm start` creates uncompressed files ?**

**.parcelrc file**:
```
{
    "extends": "@parcel/config-default",
    "compressors": {
        "*.{html,css,js,svg,map}": [
            "@parcel/compressor-gzip"
        ]
    }
}
```
**package.json**:
```
{
  "name": "interface",
  "version": "1.0.0",
  "description": "",
  "default": "index.html",
  "scripts": {
    "start": "parcel index.html --dist-dir ./dist",
    "build": "parcel build index.html --dist-dir ./build --no-source-maps"
  },
  "author": "George",
  "license": "ISC",
  "devDependencies": {
    "parcel": "^2.3.2"
  },
  "dependencies": {
    "@parcel/compressor-gzip": "^2.3.2",
    "toastify-js": "^1.11.2"
  }
}
```

With the above configuration the `npm start` command doesn't create any file in the `dist` folder and `npm run build` command creates only gzipped files in the `build` folder (which is the desired behavior).

I know that I can add `"...",` inside **.parcelrc** file so that the `npm start` command will create the uncompressed files, but that also modifies the `npm run build` command which will creates both compressed and uncompressed files, making a big clutter of files.

**Comment by mischnic** [comment, marked as answer]
This is the default behaviour: https://github.com/parcel-bundler/parcel/pull/7510, which was also released before your post. Not sure why it was behaving like that for you
